src/app/Http/Controllers/SupportPageController.php

```diff
@@ -39,15 +39,18 @@ public function show(): View
             );
         }
 
+        $path = preg_replace("/[^0-9.]/", '', base_path());
```

Comment:
This is a very expensive way of getting the current build. Use strrpos and substr instead.

Reply:
Not a good variable name